src/transformers/models/character_bert/configuration_character_bert.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+# Copyright Hicham EL BOUKKOURI, Olivier FERRET, Thomas LAVERGNE, Hiroshi NOJI,
+# Pierre ZWEIGENBAUM, Junichi TSUJII and The HuggingFace Inc. team.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" CharacterBERT model configuration """
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+CHARACTER_BERT_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "helboukkouri/character-bert": "https://huggingface.co/helboukkouri/character-bert/resolve/main/config.json",
+    "helboukkouri/character-bert-medical": "https://huggingface.co/helboukkouri/character-bert-medical/resolve/main/config.json",
+    # See all CharacterBERT models at https://huggingface.co/models?filter=character_bert
+}
+
+
+class CharacterBertConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a :class:`~transformers.CharacterBertModel`.
+    It is used to instantiate an CharacterBERT model according to the specified arguments, defining the model
+    architecture. Instantiating a configuration with the defaults will yield a similar configuration to that of
+    the CharacterBERT `helboukkouri/character-bert <https://huggingface.co/helboukkouri/character-bert>`__ architecture.
+
+    Configuration objects inherit from  :class:`~transformers.PretrainedConfig` and can be used
+    to control the model outputs. Read the documentation from  :class:`~transformers.PretrainedConfig`
+    for more information.
+
+
+    Args:
+        character_embeddings_dim (:obj:`int`, `optional`, defaults to :obj:`16`):
+            The size of the character embeddings.
+        cnn_activation (:obj:`str`, `optional`, defaults to :obj:`"relu"`):
+            The activation function to apply to the cnn representations.
+        cnn_filters (:obj:`list(list(int))`, `optional`, defaults to :obj:`[
+           [1, 32], [2, 32], [3, 64], [4, 128], [5, 256], [6, 512], [7, 1024]]`):
```

Comment:
```suggestion
        cnn_filters (:obj:`list(list(int))`, `optional`, defaults to :obj:`[[1, 32], [2, 32], [3, 64], [4, 128], [5, 256], [6, 512], [7, 1024]]`):
```
Will break the docs otherwise.